framework/web/View.php

```diff
@@ -548,4 +550,20 @@ protected function renderBodyEndHtml($ajaxMode)
 
         return empty($lines) ? '' : implode("\n", $lines);
     }
+
+    /**
+     * Append timestamp if the url starts with the alias 'web' and the configuration is turned on
+     *
+     * @param $url
+     * @return string
+     */
+    public function appendTimestamp($url)
+    {
+        if (Yii::$app->assetManager->appendTimestamp && strncmp($url, '@web/', 5) === 0) {
```

Comment:
I think the first part of this if should be outside this function and be called on the `registerCssFile` and `registerJsFile` instead. As @githubjeka I might want to use this method for some file and not for others. `appendTimestamp` is the default behavior but it should be possible to use it file by file